packages/myst-cli/src/build/build.ts

```diff
@@ -26,29 +27,33 @@ export type BuildOpts = {
 };
 
 export function hasAnyExplicitExportFormat(opts: BuildOpts): boolean {
-  const { docx, pdf, tex, xml, md, meca } = opts;
-  return docx || pdf || tex || xml || md || meca || false;
+  const { docx, pdf, tex, typst, xml, md, meca } = opts;
```

Comment:
We just need to refactor this boilerplate somehow to reduce repetition. Some way to specify things declaratively only once